docfx/en/components/toc.json

```diff
@@ -1332,6 +1332,12 @@
     "href": "inputs/ripple.md",
     "status": "NEW"
   },
+  {
+    "exclude": ["Angular", "React"],
+    "name": "Slider & Range Slider {ReleaseStatus_WebsInputs}",
+    "href": "inputs/slider.md",
+    "status": "NEW"
```

Comment:
@DiyanDimitrov should this new topic be excluded for Blazor? I asked we currently do not have Blazor slider and samples for it here:
https://github.com/IgniteUI/igniteui-blazor-examples/tree/vnext/samples/inputs